draft-ietf-lake-edhoc.md

```diff
@@ -1744,17 +1744,20 @@ The columns of the registry are Name, Label, Description, and Reference, where L
 IANA is requested to register the following entries in the "COSE Header Parameters" registry under the registry group "CBOR Object Signing and Encryption (COSE)" (see {{fig-header-params}}): The value of the 'kcwt' header parameter is a COSE Web Token (CWT) {{RFC8392}}, and the value of the 'kccs' header parameter is a CWT Claims Set (CCS), see {{term}}. The CWT/CCS must contain a COSE_Key in a 'cnf' claim {{RFC8747}}. The Value Registry for this item is empty and omitted from the table below.
 
 ~~~~~~~~~~~ aasvg
-+-----------+-------+----------------+---------------------------+
-| Name      | Label | Value Type     | Description               |
-+===========+=======+================+===========================+
-| kcwt      | TBD1  | COSE_Messages  | A CBOR Web Token (CWT)    |
-|           |       |                | containing a COSE_Key in  |
-|           |       |                | a 'cnf' claim             |
-+-----------+-------+----------------+---------------------------+
-| kccs      | TBD2  | map / #6(map)  | A CWT Claims Set (CCS)    |
-|           |       |                | containing a COSE_Key in  |
-|           |       |                | a 'cnf' claim             |
-+-----------+-------+----------------+---------------------------+
++------+-------+---------------+------------------------------------+
```

Comment:
Looks good.

(Is there a particular reason you are not using a table here?
Just curious.)

Reply:
> Is there a particular reason you are not using a table here
No, no particular reason. None of the "tables" in the document are real tables. I thought about fixing it before when I added aasvg but did not have the time. Maybe time to fix that for -21